meteor/client/ui/Settings/Studio/Routings/RouteSets.tsx

```diff
@@ -391,6 +408,28 @@ function RouteSetRow({
 								</button>
 							)}
 						</div>
+						<RouteSetAbPlayers routeSet={routeSet} overrideHelper={overrideHelper} />
+						<div className="mod">
+							<button className="btn btn-primary right" onClick={() => toggleExpanded(routeSet.id)}>
```

Comment:
UX: I think the position of this collapse button is confusing.
It being on the same row as the 'add ab' button, suggests it is related to that. probably enough to put it in a row below that button?